force-app/nksAddresses/lwc/nksTemporaryAddress/nksTemporaryAddress.js

```diff
@@ -24,44 +33,98 @@ export default class NksBostedAddress extends LightningElement {
         }
     }
 
+    /* Function to handle open/close section */
+    handleOpen() {
+        if (this.sectionClass === 'slds-section section slds-is-open') {
+            this.sectionClass = 'slds-section section';
+            this.sectionIconName = 'utility:chevronright';
+            this.isExpanded = false;
+            this.ariaHidden = true;
+        } else {
+            this.sectionClass = 'slds-section section slds-is-open';
+            this.sectionIconName = 'utility:chevrondown';
+            this.isExpanded = true;
+            this.ariaHidden = false;
+        }
+    }
```

Comment:
bare som forslag: 
`handleOpen() {
    this.isExpanded = !this.isExpanded;
    this.sectionClass = this.isExpanded 
        ? 'slds-section section slds-is-open' 
        : 'slds-section section';
    this.sectionIconName = this.isExpanded 
        ? 'utility:chevrondown' 
        : 'utility:chevronright';
    this.ariaHidden = !this.isExpanded;
}
`